Source/Tools/FEXServer/ProcessPipe.cpp

```diff
@@ -449,6 +449,7 @@ void WaitForRequests() {
               int NewFD = accept(ServerSocketFD, reinterpret_cast<struct sockaddr*>(&Addr), &AddrSize);
 
               // Add the new client to the temporary array
+              LogMan::Msg::DFmt("[FEXServer] New connection accepted");
```

Comment:
This gets /really/ chatty when running any real workloads and will spam hundreds of messages every time pressure-vessel launches plus hundreds more each time dxsetup runs. We have configurations running FEXServer with its output log going to the journal so we want to be mindful of logging.

I am going to need to nak from this line alone, but if the other two lines were actually getting their logs out then I would do it thrice over.

Reply:
Since this is a DFmt, doesn't it mean that you need to enable Debug messaging for it to actually spam the logs? I would expect that for normal runs, the log level would be info or something like that. I don't think Debug logging in general should be enabled for production executation.

Reply:
Log levels don't mean anything in FEX's context atm, they were more a qualifier for the type of log than anything else atm. FEXInterpreter will write them all, and FEXServer will display them all regardless.

Reply:
Ufff - I didn't know that. :-/
That's unfortunate. So, I guess the issue is that we always set the log level to debug? We sometimes do use IFmt and other times DFmt...